templates/service.yaml

```diff
@@ -12,5 +12,6 @@ spec:
       protocol: TCP
       name: http
   selector:
+    matchLabels:
```

Comment:
There seems to be no `matchLabels` spec in a Kubernetes Service: https://kubernetes.io/docs/concepts/services-networking/service/

Reply:
Yes you are right. I thought that I reverted this change prior to committing.

Reply:
Helm Basics lab 5 also contains this line, maybe it comes from there.